src/qt/ui/modalview.ui

```diff
@@ -95,40 +95,81 @@
       <x>-1</x>
       <y>-1</y>
       <width>401</width>
-      <height>359</height>
+      <height>442</height>
```

Comment:
Is the height change intentional?
Often QtCreator fiddles with that without really changing it.